### DISCUSSION TITLE
Where to offer wiki / doc

### DISCUSSION THREAD

**Discussion by PhillypHenning** [bitovi/bitops#205] [Community]
# The Questions
Where should our wiki / doc be hosted that will make it easily accessible by users and bitops developers.
What information should go there? 
Who should maintain the pages? 
Who should add new pages?

**Comment by ConnorGraham** [comment]
Wiki = docs for developing
Website = docs for using

We'll probably want an issue to re-work this site page accordingly. Until then, I think all docs for developing can continue to be dumped within it
https://github.com/bitovi/bitops/blob/master/docs/development/development.md

**Reply by PhillypHenning** [reply]
Just to clarify you're suggesting that any developer related doc is put into the [development docs folder](https://github.com/bitovi/bitops/tree/master/docs/development)

**Reply by ConnorGraham** [reply]
Yes, but only until this discussion and potential follow up issue is resolved. 

Long term I am advocating developer documentation goes in the GitHub wiki

**Comment by mickmcgrath13** [comment]
- Internal slack only for client based bitops discussions and messages about internal docs access
- bitovi community slack for all day to day activity
  - PR reviews
  - questions about implementation, errors, etc
  - random thoughts
  - general help
- bitops repo github discussions for ideas we want to pursue
  - create a discussion and then provide the link in the community slack to ask for feedback
  - possibly cross-link the post in community slack to the internal slack to help with awareness
  - when we finalize and idea, the results will be turned into one of the following
    - GH issue to implement functionality
    - GH issue to implement docs
    - wiki page/update
- Bitops repo Wiki for developer docs
  - developing bitops and plugins, release patterns, etc
- BitOps docs for users
- Bitovi Confluence
  - internal tracking
  - use less over time in favor of public-ness
